src/test/java/com/codurance/retropolis/controllers/BoardControllerTest.java

```diff
@@ -95,11 +100,67 @@ void returns_board_with_columns_and_cards() throws Exception {
     assertEquals(userName, cardResponse.getUsername());
   }
 
+  @Test
+  void returns_a_new_board() throws Exception {
+    NewBoardRequestObject requestObject = new NewBoardRequestObject(BOARD_TITLE, USER_EMAIL);
+    given(boardService.createBoard(any(NewBoardRequestObject.class)))
+        .willReturn(new Board(BOARD_ID, BOARD_TITLE, Collections.emptyList()));
+
+    MvcResult response = mockMvc.perform(MockMvcRequestBuilders.post(URL)
+        .content(asJsonString(requestObject))
+        .contentType(MediaType.APPLICATION_JSON)
+        .accept(MediaType.APPLICATION_JSON))
+        .andExpect(status().isCreated()).andReturn();
+
+    String responseBody = response.getResponse().getContentAsString();
+    Board boardResponse = objectMapper.readValue(responseBody, new TypeReference<>() {
+    });
+
+    assertEquals(BOARD_TITLE, boardResponse.getTitle());
+    assertEquals(BOARD_ID, boardResponse.getId());
+  }
+
+  @Test
+  void returns_bad_request_when_title_is_empty() throws Exception {
+    NewBoardRequestObject requestObject = new NewBoardRequestObject("", "john.doe@codurance.com");
+    String content = asJsonString(requestObject);
+    List<String> errorResponse = performHttpPostRequest(content, status().isBadRequest());
+    Assert.assertEquals("Title must not be less than 1 character", errorResponse.get(0));
+  }
+
+  @Test
+  void returns_bad_request_when_title_is_null() throws Exception {
+    List<String> errorResponse = performHttpPostRequest("{\"userEmail\":\"john.doe@codurance.com\"}",
+        status().isBadRequest());
+    Assert.assertEquals("Title cannot be empty", errorResponse.get(0));
+  }
+
   private Board requestBoard() throws Exception {
-    MvcResult httpResponse = mockMvc.perform(MockMvcRequestBuilders.get(URL).header(HttpHeaders.AUTHORIZATION, TOKEN))
+    MvcResult httpResponse = mockMvc
+        .perform(MockMvcRequestBuilders.get(SPECIFIC_BOARD_URL).header(HttpHeaders.AUTHORIZATION, TOKEN))
         .andExpect(status().isOk()).andReturn();
     String contentAsString = httpResponse.getResponse().getContentAsString();
     return objectMapper.readValue(contentAsString, new TypeReference<>() {
     });
   }
+
+  @Test
+  void returns_bad_request_when_userEmail_is_invalid() throws Exception {
```

Comment:
not sure why this test is in between two private helper functions. i'd move all the private functions together, or move the helpers below the set of tests they help